### PR TITLE
[tune] Support numpy types in TBXlogger

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?
Numpy types in the hyperparameter space seem to be dropped. This PR aims to scrub it properly.

## Related issue number

https://github.com/ray-project/tune-sklearn/issues/201

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/tune/tests/test_logger.py [hunk 1]
```diff
@@ -169,8 +169,11 @@ def testTBX(self):
                     "D": 123
                 }
             },
-            "d": np.int64(1),
-            "e": np.bool8(True)
+            "int32": np.int32(1),
+            "int64": np.int64(1),
+            "bool8": np.bool8(True),
+            "float32": np.float32(1),
+            "float64": np.float64(1),
```
Comment (krfricke) [inline python/ray/tune/tests/test_logger.py:176]: Do we test/check the logged output somewhere?

Comment (richardliaw) [reply python/ray/tune/tests/test_logger.py]: I just added a test, take a look?

**Review by krfricke — APPROVED** [review]
Looks good to me

**Comment by richardliaw** [comment]
Test failures unrelated

**Pull request merged**